pkg/endpoint/l3maps.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2016-2017 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package endpoint
+
+import (
+	"os"
+
+	"github.com/cilium/cilium/pkg/maps/cidrmap"
+)
+
+type L3MapType int
+
+const (
+	IPv6Ingress L3MapType = iota
+	IPv4Ingress
+	IPv6Egress
+	IPv4Egress
+	MapCount
+)
+
+type L3Maps [MapCount]*cidrmap.CIDRMap
+
+func (l3 *L3Maps) DestroyBpfMap(mt L3MapType, path string) {
+	if l3[mt] != nil {
+		l3[mt].Close()
+		l3[mt] = nil
+	}
+	os.RemoveAll(path)
+}
+
+func (l3 *L3Maps) CreateBpfMap(mt L3MapType, path string) error {
```

Comment:
Should we rename this to `ResetBpfMap`? I was initially confused that this would be a one-time operation.